11_hexclover.md

```diff
@@ -1,28 +0,0 @@
-## Hex of Clovers
```

Comment:
This is a subsection in chapter 11, so for organizational purposes  I have moved this to the end of 11_big_patterns.md